gen-vinstr.sh

```diff
@@ -0,0 +1,15 @@
+CMAKE_SOURCE_DIR=$1
```

Comment:
Can you create a `scripts/`directory in tilt repo and move this file inside that?